docs/metrics/README.md

```diff
@@ -31,7 +31,8 @@ MyFruitCounter.Add(7, new("color", "red"), new("name", "apple"), new("taste", "s
   performance. Using
   [`TagList`](https://learn.microsoft.com/dotnet/api/system.diagnostics.taglist?view=net-7.0#remarks)
   avoids allocating any memory for up to eight tags, thereby, reducing the
-  pressure on GC to free up memory.
+  pressure on GC to free up memory. 
+- DO NOT use `TagList` for `<=3 tags` as that would lead to poorer performance than individually emitting the tags.
```

Comment:
Consider rephrasing the entire paragraph to make the flow more intuitive, for example:

For the best performance, it is highly recommended to pass in tags in certain ways so allocations are only happening on the stack rather than the heap, which eliminates pressure on the GC (garbage collector):
* When reporting measurements with `3` tags or less, use ...
* When reporting measurements with more than `3` tags, use ...

Reply:
Good suggestions!

Fix this.